src/Components/Header/Header.stories.jsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { React, useState } from 'react'
+import Header from './Header'
+
+export default {
+    title: 'Header component',
+    component: Header
+    // argTypes: {
+    //   background: { control: "color" },
+    //   active: { control: 'boolean' }
+    // },
+}
+
+// cia aprasomas sablonas kaip bus atvaizduojamas komponentas:
+// propsus paduodame i f-ja kaip argumentus (args) visus iskart arba pavieniuis(uzkomentuotos eilutes)
+const Template = (args) => (
+// const Template = ({color, background}}) => {
+
+    <div>
+        <Header {...args} />
+        {/* <StoryBookExampleComponent color={color} background={background} /> */}
+    </div>
+)
+
+// paduodamos skirtingos argumentu kombinacijos storiams, kurie naudoja ta sablona:
+export const Arguments = Template.bind({})
+Arguments.args = {
+    primary: true,
+    fontSize: '2rem',
+    fontFamily: 'serif',
+    color: 'white',
+    paddingTop: '1%',
+    paddingBottom: '5%',
+    height: '55px',
+    width: '55px'
+}
+// naudojami dekoratoriai
```

Comment:
Why we have commented code here?

Reply:
I took all Evelina's pattern here and adopted to my styles and component. When we reviewed this code with her, she explained me that here are shown several different approaches in examples. So, I commented all what was not used for my Header.stories creation. I thinc for Hoc layout or wrapper also can be used pattern from line 54... to extend already assigned styles.
Should I delete this commented part or leave as is?

Reply:
I think we should delete all unecessary code, and avoid to leave unused code. You can make a copy to your self.

Reply:
After you'll remove unused code, please merge this PR ;)

Reply:
Done :)